server/src/main/java/com/broadcom/lsp/cobol/service/CopybookServiceImpl.java

```diff
@@ -153,6 +148,34 @@ public void observerCallback(DataEvent event) {
     }
   }
 
+  private CopybookModel emptyModel(@NonNull String copybookName) {
```

Comment:
This method can be moved to `CopybookModel` class

Reply:
Agree. Or we can have a constructor with only one parameter: copybookName.

Reply:
That's true. But I think that due to `CopybookServiceImpl` is the only class that creates copybook models, an empty model is a part of its internal logic, not of `CopybookModel` itself.